lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -37,31 +49,69 @@ provider:
     - Effect: 'Allow'
       Action: 'kms:Decrypt'
       Resource:
-        - "arn:aws:kms:#{AWS::Region}:#{AWS::AccountId}:key/efba9b4c-8e64-430c-86c9-f00eaf69e582"
+        - "arn:aws:kms:#{AWS::Region}:#{AWS::AccountId}:key/eb628cf2-33db-4909-8581-d9c0626ac8ec"
```

Comment:
We should look into the different deployment options in _serverless_, so that we can have both dev and prod managed from the one configuration file, instead of overwriting ourselves in git all the time.